tonic/src/transport/server/tls.rs

```diff
@@ -46,7 +48,19 @@ impl ServerTlsConfig {
         }
     }
 
+    /// Per session TLS secrets will be written to a file given by the SSLKEYLOGFILE environment variable.
+    pub fn install_key_log_file(self, install_key_log_file: bool) -> Self {
```

Comment:
Is this something that is dangerous and we'd want to put behind a dangerous feature flag?